templates/Register.html

```diff
@@ -23,22 +26,114 @@
         <input name="spotifytoken" id="spotify" type="checkbox">
     </p>
 
+
     <Label for="pin">4 character pin: </Label>
     <input name="pin" type="text" Placeholder="****" required pattern=".{4,4}" maxlength="4" minlength="4">
     <br>
     <p>Picture of face:</p>
-    <input name="facepath"  type="file" accept="image/*" capture="camera" required>
+    <input name="facepath" id="facepath"  type="file" accept="image/*" capture="camera" required>
 
     <hr/>
-
-    <input class="btn btn-lg btn-success" type="submit">
+    <!--can't get to post the way I want currently
+    "time();calendar(); spotifytoken(); googletoken(); maps(); pin(); weather();twittertoken(); name();email(); facepath(); postIt(); printC()"-->
+    <input onclick= "test()" class="btn btn-lg btn-success" type="submit">
     <input class ="btn btn-lg btn-danger" type="button" onclick="window.location.href='{{url_for('index')}}'" value="Cancel" />
 </form>
 
 <form action="{{url_for('logout')}}">
 
 </form>
 </body>
+<!--not sure how to call token-->
+<script>
```

Comment:
Put this JS code in the JavaScript folder.

Reply:
in its own file

Reply:
I thought you said we couldn't do relative linking? Or is js different than css?

Reply:
you cant do relative linking but you can do this  {{url_for('logout')}}